Sources/IO/Geometry/GLTFReader/Reader.js

```diff
@@ -0,0 +1,526 @@
+/* eslint-disable no-case-declarations */
```

Comment:
A "Reader" in the VTK world is not meant to modify a scene (i.e. modify renderer, create actors...) but only produce data objects. Such "modifying scene" role is instead given to an "Importer" class.

See `https://vtk.org/doc/nightly/html/classvtkGLTFImporter.html`